op-wheel/cheat/cheat.go

```
@@ -379,6 +379,13 @@ func SetBalance(addr common.Address, amount *big.Int) HeadFn {
 	}
 }
 
+func SetCode(addr common.Address, code string) HeadFn {
+	return func(headState *state.StateDB) error {
+		headState.SetCode(addr, common.FromHex(code))
```

it should `code []byte`, otherwise it's going to panic on invalid strings